### ISSUE TITLE
[CI][Fp32][Vega20] Hip All Stage timeout Four Times in a Row 

### ISSUE THREAD

**Issue by junliume** [ROCmSoftwarePlatform/MIOpen#1697]
[Fp32 Hip All Vega20 Log2.log](https://github.com/ROCmSoftwarePlatform/MIOpen/files/9338960/Fp32.Hip.All.Vega20.Log2.log)
[Fp32 Hip All Vega20 Log1.log](https://github.com/ROCmSoftwarePlatform/MIOpen/files/9338961/Fp32.Hip.All.Vega20.Log1.log)

Tests timeout after 5 hours

Establish this ticket to track diagnostic processes on it

**Comment by junliume** [comment]
@atamazov @shurale-nkn  if you are interested in taking a look

**Comment by junliume** [comment]
Cannot reproduce on a gfx906 node with 64 CU
docker:
```
compute-artifactory.amd.com:5000/rocm-plus-docker/framework/compute-rocm-dkms-no-npi-hipclang:10584_ubuntu20.04_py3.7_pytorch_rocm5.3_internal_testing_9b13302 
```
base OS has ROCM 5.3.0

**Comment by junliume** [comment]
CC: @aska-0096 
This issue is caused by a large test `test_miopendriver_big_tensor`. It's hard to reproduce in dev env but indeed stuck at our CI pipelines for a few outdated ASICs.

**Comment by junliume** [comment]
Should be resolved by #1693
